### ISSUE TITLE
Implement Dark Theme

### ISSUE THREAD

**Issue by CodyMichaelSimmons** [CodyMichaelSimmons/CodyMichaelSimmons.github.io#17]
Theme should be predominantly dark while maintaining ease of accessibility. This feature should be toggle able.
